File: Kogebog.API/Kogebog.API.Repository/Database/Entities/Unit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kogebog.API.Repository.Database.Entities
{
    public class Unit
    {
        public Guid Id { get; set; }
        public required string Name { get; set; }
        public required string PluralName { get; set; }
        public required string Abbreviation { get; set; }
    }
}

File: Kogebog.App/kogebog_flutter/lib/models/unit.dart
class Unit {
  final String id;
  final String name;
  final String pluralName;
  final String abbreviation;

  Unit({required this.id, required this.name, required this.pluralName, required this.abbreviation});

  factory Unit.fromJson(Map<String, dynamic> json) {
    return Unit(id: json['id'], name: json['name'], pluralName: json['pluralName'], abbreviation: json['abbreviation']);
  }
}